src/plugins/data_table/row.js

```diff
@@ -10,8 +10,16 @@ var Row = (function() {
 
     render : function() {
       var cells = this.findCells(), node;
-      _.each(this.columnsConfig, function(config) {
-        node = cells.filter(config.nodeMatcher(config));
+      var cleanConfigs = this.columnsConfig.map(function(currentValue,index){
+        if (currentValue.bulk || ((currentValue.attr || currentValue.title) && currentValue.visible) ) {
+          return currentValue;
+        }
+      }).filter(function(currentValue) {
+         return currentValue !== undefined;
+        }
```

Comment:
is this filter necessary? not following how an undefined config would get passed back from the map function